models/demos/llama3/demo/simple_vision_demo.py

```diff
@@ -42,7 +43,22 @@ def sample(logits):
     return sample
 
 
-def create_multimodal_model(mesh_device, max_batch_size, max_seq_len, dtype=ttnn.bfloat16):
+def get_batch_sampler(temperature, top_p, tokenizer):
+    def sample(logits):
+        if temperature > 0:
+            probs = torch.softmax(logits[:, -1] / temperature, dim=-1)
+            next_token = llama_reference_generation.sample_top_p(probs, top_p)
+        else:
+            next_token = torch.argmax(logits[:, -1], dim=-1)
+
+        next_tokens = next_token.reshape(-1)
+        texts = [tokenizer.decode([next_tokens[i].item()]) for i in range(len(next_tokens))]
+        return next_tokens, texts
+
+    return sample
+
+
+def create_multimodal_model(mesh_device, max_batch_size, max_seq_len, dtype=ttnn.bfloat16, use_paged_kv_cache=False):
```

Comment:
I removed `test_llama_vision_model` because it is broken and not useful right now -- we will use real benchmarks to get accuracy info for llama vision.